src/components/explore/ToplineMetrics.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+import Tweenable from 'udgl/data-graphics/motion/Tweenable.svelte';
+import ToplineRow from './ToplineRow.svelte';
+
+import { toplineRefLabel } from '../../utils/constants';
+
+import {
+  formatCount, formatSignCount,
+  formatParenPercent,
+} from '../../utils/formatters';
+
+export let reference;
+export let hovered;
+export let showHover;
+export let aggregationLevel;
+
+function absDiff(a, b, perc = false) {
+  return (a - b) / (perc ? b : 1);
+}
+
+</script>
+
+  <div style='
+  padding-left: {toplineRefLabel.left - toplineRefLabel.icon}px;
```

Comment:
Setting the dynamic padding inline here is fine but the rest of the styles should eiher be in `<style>` or  `global.css`.